MyXamarinApp.API/Repository/EmployeeRepository.cs

```diff
@@ -39,17 +46,30 @@ public async Task<EmployeeModel> AddEmployee(EmployeeModel employeeModel)
             var employee = _mapper.Map<Employee>(employeeModel);
             _context.Employees.Add(employee);
             await _context.SaveChangesAsync();
-            return employeeModel;
+            return _mapper.Map<EmployeeModel>(employee);
+        }
+
+        /// <summary>
+        /// Updates the employee details to the Employee table.
+        /// </summary>
+        /// <param name="employeeModel">Employee to be added.</param>
+        /// <returns>Employee model of the new Employee.</returns>
+        public async Task<EmployeeModel> UpdateEmployee(EmployeeModel employeeModel)
+        {
+            var employee = _mapper.Map<Employee>(employeeModel);
+            _context.Employees.Update(employee);
+            await _context.SaveChangesAsync();
+            return _mapper.Map<EmployeeModel>(employee);
```

Comment:
Although technically this is the same, I would expect the value that is actually in the database to be returned back. So when you update something into the database, you need to fetch that data back from database and then send it, if you really want to sent it